content/acra/acra-in-depth/security-design/threat-models-and-guarantees.md

```diff
@@ -80,6 +84,11 @@ By differentiating the sensitive data via [Zones](/acra/security-controls/zones/
 
 - Zone IDs are used as context when encrypting data using Themis Secure Cell, providing additional protection.
 
+{{< hint warning >}}
+Zones are deprecated since 0.94.0, will be removed in 0.95.0. 
+AcraEnterprise provides approach of differentiating ClientID dynamically via SQL.
```

Comment:
```suggestion
[Acra Enterprise Edition](https://docs.cossacklabs.com/acra/enterprise-edition/) provides a better approach of differentiating ClientID dynamically via SQL.
```

Reply:
can we write couple of words about "differentiating ClientID dynamically" somewhere in the docs? Describe how it works, what it requires, and point to the [Acra Enterprise Edition](https://docs.cossacklabs.com/acra/enterprise-edition/).

Reply:
As for me, it makes sense to create a separate PR that contains only info about dynamic ClientID extracting and not mix in one PR.

Reply:
well, agreed, let's put it to a different PR. It's just that we have mentioned dynamic ClientID extracting here, so I suggested to add more details. I'm totally OK with doing it in a separate PR

Reply:
Okay, lets add it in separate PR. I mentioned it here because there are several ways of deprecating zones. Remove this paragraph, mark as deprecated and mark + suggest a replacement. Because it looks like we describe that it is cool way to increase security with zones and then we deprecate it )) 

But we can just remove mentioning zones here.

Reply:
Well, I don't think that we should remove mentioning zones completely, at least until we don't have separate Acra docs for each version. Image that some of our customers already use Zones and old Acra. They might need info about zones as they use it now.